plugins/rbw/rbw.plugin.zsh

```diff
@@ -17,3 +17,23 @@ if [[ ! -f "$ZSH_CACHE_DIR/completions/_rbw" ]]; then
 fi
 
 rbw gen-completions zsh >| "$ZSH_CACHE_DIR/completions/_rbw" &|
+
+# rbwpw function copies the password of a service to the clipboard
+# and clears it after 20 seconds
+function rbwpw {
+  [[ $# -ne 1 ]] && echo "usage: rbwpw <service>" && return 1
+  local service=$1
+  rbw unlock || (echo "rbw is locked" && return 1)
+  local pw=$(rbw get $service 2>/dev/null)
+  [[ -z $pw ]] && echo "$service not found" && return 1
+  echo -n $pw | clipcopy
+  echo "password for $service copied!"
+  (sleep 20 && clipcopy </dev/null 2>/dev/null) &|
+}
+
+function _rbwpw {
+  local -a services=("${(@f)$(rbw ls 2>/dev/null)}")
```

Comment:
Inline declaration and assigning of an array breaks in older zsh versions (can't remember at which point it starts failing). You should do this in two lines:
```suggestion
  local -a services
  services=("${(@f)$(rbw ls 2>/dev/null)}")
```